auto_submit/lib/requests/github_webhook.dart

```diff
@@ -17,7 +18,26 @@ class GithubWebhook {
   Future<Response> post(Request request) async {
     final Map<String, String> reqHeader = request.headers;
     logger.info('Header: $reqHeader');
+
+    //Listen to the pull request with 'autosubmit' label.
+    bool hasAutosubmit = false;
     final String rawBody = await request.readAsString();
+    final body = json.decode(rawBody) as Map<String, dynamic>;
+    List<Map<String, dynamic>> labels = <Map<String, dynamic>>[];
+    if (body.containsKey('pull_request') && body['pull_request'].containsKey('labels')) {
```

Comment:
Avoid using dynamic. Cocoon relies on https://pub.dev/packages/github for the data models, so in this case you should use https://pub.dev/documentation/github/latest/github/IssueLabel/IssueLabel.fromJson.html

1. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-var-and-dynamic